question/bank/viewquestionname/templates/questionname_display.mustache

```diff
@@ -0,0 +1,60 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_viewquestionname/questionname_display
+
+    Example context (json):
+    {
+        "displaydata": [
+          {
+                "haslabel": false,
+                "nameclasses": "class-name class-name",
+                "namecontent": "Question name",
+                "idnumber": "questionidnumber",
+                "tags": "tags html",
+                "labelfor": "labelkey"
+          }
+        ]
+    }
+}}
+
+{{^haslabel}}
+    <span class="d-inline-flex flex-nowrap overflow-hidden w-100">
+        <span class="{{nameclasses}}">
+            {{namecontent}}
+        </span>
+        <span class="ml-1">
+            <span class="accesshide">{{#str}}idnumber, question{{/str}}</span>&nbsp<span class="badge badge-primary">{{idnumber}}</span>
+        </span>
+    </span>
+    <span class="d-inline-flex flex-nowrap overflow-hidden w-100">
+        {{{tags}}}
+    </span>
+{{/haslabel}}
+{{#haslabel}}
+    <label for="{{labelfor}}" class="d-inline-flex flex-nowrap overflow-hidden w-100">
+        <span class="questionname flex-grow-1 flex-shrink-1 text-truncate">
+            {{namecontent}}
+        </span>
+        <span class="ml-1">
+            <span class="accesshide">{{#str}}idnumber, question{{/str}}</span><span class="badge badge-primary">{{idnumber}}</span>
+        </span>
+    </label>
+    <label for="{{labelfor}}" class="d-inline-flex flex-nowrap overflow-hidden w-100">
```

Comment:
@safatshahin can you please add a `{{#tags}}` before this line here so we won't see the question like this:

![image](https://user-images.githubusercontent.com/6785838/120413741-3798b000-c39c-11eb-8e76-923f244f40bb.png)

Also the close one `{{/tags}}` at the end of `</label>`